tests/test_instcat_parser.py

```diff
@@ -289,7 +289,7 @@ def test_object_extraction_galaxies(self):
                                    truth_data['minorAxis'][i_obj], 13)
             self.assertAlmostEqual(gs_obj.majorAxisRadians,
                                    truth_data['majorAxis'][i_obj], 13)
-            self.assertAlmostEqual(gs_obj.positionAngleRadians,
+            self.assertAlmostEqual(2.*np.pi - gs_obj.positionAngleRadians,
```

Comment:
Angles are defined counter-clockwise?

Reply:
I don't think it matters.  The upshot of this change is that the PA sign is flipped and 2\*pi is added to get it back in the (0, 2\*pi) range.